prov/verbs/src/verbs_cq.c

```diff
@@ -432,20 +432,22 @@ int fi_ibv_cq_open(struct fid_domain *domain, struct fi_cq_attr *attr,
 		goto err3;
 	}
 
-	_cq->cq = ibv_create_cq(_cq->domain->verbs, attr->size, _cq,
-				_cq->channel, attr->signaling_vector);
-	if (!_cq->cq) {
-		ret = -errno;
-		goto err3;
-	}
-
-	if (_cq->channel) {
-		ret = ibv_req_notify_cq(_cq->cq, 0);
-		if (ret) {
-			FI_WARN(&fi_ibv_prov, FI_LOG_CQ, "ibv_req_notify_cq failed\n");
-			goto err4;
-		}
-	}
+    if (attr->format != FI_CQ_FORMAT_TAGGED) {
+        _cq->cq = ibv_create_cq(_cq->domain->verbs, attr->size, _cq,
+                    _cq->channel, attr->signaling_vector);
+        if (!_cq->cq) {
+            ret = -errno;
+            goto err3;
+        }
+
+        if (_cq->channel) {
+            ret = ibv_req_notify_cq(_cq->cq, 0);
+            if (ret) {
+                FI_WARN(&fi_ibv_prov, FI_LOG_CQ, "ibv_req_notify_cq failed\n");
+                goto err4;
+            }
+        }
+    }
```

Comment:
The indentation is off for this change.  Please use tabs, not spaces, for indentation, with the tab stop set to 8.

Separate from that, the behavior should key off of the domain (e.g. domain->rdm field), not the CQ attribute format.  Though we may want to add checks such that if domain->rdm is set, then only the tagged format is supported.